programs/vote/benches/cu_vote.rs

```diff
@@ -0,0 +1,165 @@
+        let instruction_accounts = vec![
+            AccountMeta {
+                pubkey: vote_pubkey,
+                is_signer: false,
+                is_writable: true,
+            },
+            AccountMeta {
+                pubkey: sysvar::rent::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+            AccountMeta {
+                pubkey: sysvar::clock::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+            AccountMeta {
+                pubkey: node_pubkey,
+                is_signer: true,
+                is_writable: false,
+            },
+        ];
+
+        let _accounts = Self::process_instruction(
+            &instruction_data,
+            vec![
+                (vote_pubkey, vote_account),
+                (sysvar::rent::id(), create_default_rent_account()),
+                (sysvar::clock::id(), create_default_clock_account()),
+                (node_pubkey, node_account),
+            ],
+            instruction_accounts,
+            Ok(()),
+        );
+    }
+    fn test_authorize_voter(&self) {
+        let (vote_pubkey, vote_account) = create_test_account();
+        let authorized_voter_pubkey = solana_pubkey::new_rand();
+        let clock = Clock {
+            epoch: 1,
+            leader_schedule_epoch: 2,
+            ..Clock::default()
+        };
+        let clock_account = account::create_account_shared_data_for_test(&clock);
+        let instruction_data = serialize(&VoteInstruction::Authorize(
+            authorized_voter_pubkey,
+            VoteAuthorize::Voter,
+        ))
+        .unwrap();
+        let transaction_accounts = vec![
+            (vote_pubkey, vote_account),
+            (sysvar::clock::id(), clock_account),
+            (authorized_voter_pubkey, AccountSharedData::default()),
+        ];
+        let instruction_accounts = vec![
+            AccountMeta {
+                pubkey: vote_pubkey,
+                is_signer: true,
+                is_writable: true,
+            },
+            AccountMeta {
+                pubkey: sysvar::clock::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+        ];
+
+        let _accounts = Self::process_instruction(
+            &instruction_data,
+            transaction_accounts,
+            instruction_accounts,
+            Ok(()),
+        );
+    }
+}
+
+fn bench_initialize_account(c: &mut Criterion) {
+    let test_setup = TestSetup::new();
+    c.bench_function("vote_instruction_initialize_account", |bencher| {
+        bencher.iter(|| test_setup.test_initialize_account())
+    });
+}
+
+fn bench_authorize(c: &mut Criterion) {
+    let test_setup = TestSetup::new();
+    c.bench_function("vote_instruction_authorize", |bencher| {
+        bencher.iter(|| test_setup.test_authorize_voter())
+    });
+}
+
+criterion_group!(benches, bench_initialize_account, bench_authorize);
```

Comment:
vote has 16-ish instructions (here: https://github.com/anza-xyz/agave/blob/master/sdk/vote-interface/src/instruction.rs#L27), will benches for the rest of them being added here?